adr/ADR-32.md

```diff
@@ -132,9 +137,9 @@ Returns a JSON having the following structure:
      */
     version: string,
     /**
-     * Subject where the service can be invoked
+     * Root subject of the service
      */
-    subject: string
+    root_subject: string
 }
```

Comment:
We should have a the root subject, and a list of all the other subjects (as augmented by additional endpoints.